src/Dependencies.js

```diff
@@ -118,7 +118,7 @@ class Dependencies {
             );
 
             if (process.env.NODE_ENV !== 'test') {
-                process.exit();
+                process.exit(126);
```

Comment:
```suggestion
                process.exit(1);
```

126 isn't appropriate here as that is an indication that something is not a valid executable — or that you don't have permission to execute it.